addons/mail/static/src/models/attachment_viewer.js

```diff
@@ -16,6 +16,19 @@ registerModel({
         containsElement(element) {
             return Boolean(this.component && this.component.root.el && this.component.root.el.contains(element));
         },
+        /**
+         * Called when clicking on mask of attachment viewer.
+         *
+         * @param {MouseEvent} ev
+         */
+        onClick(ev) {
```

Comment:
`onClick` feel too generic inside a model. Based on the comment, it's triggered when clicking on the dark background. So maybe `onClickMask`, `onClickBackground` or`onClickOutside` ?

Reply:
The handling happens when just clicking on mask/background/outside, but I would not rename the handling method because this happens because of poor usage of `stopPropagation` in `AttachmentViewer` code.
Without consideration of these `stopPropagation`, this is precisely an `onClick` handler.

I would name it properly only when the code of component is cleaned enough, i.e. no more `stopPropagation`, proper naming of part of template (there should be a `mask` part, and the `onClick` is hooked precisely on this part and not above, etc.).

So in the mean time, I much prefer keeping `onClick` name.